csfieldguide/templates/interactives/dot-combinations.html

```diff
@@ -4,135 +4,59 @@
 {% load static %}
 
 {% block html %}
-  <div class="row mb-3 uninteractive">
-    <div class="col-6 p-2 text-center">
-      <!-- Row 1 -->
-      <h5>{% trans "Two Dots" %}</h5>
+  <div class="container-fluid my-3 dot-combinations text-center">
 
-      <!-- Row 2 -->
-      <div>
-        <div class="d-inline-flex justify-content-center py-2 border-bottom">
-          <div>1</div>
-          <div class="div-circle white mx-3"></div>
-          <div class="div-circle white"></div>
-        </div>
-      </div>
-
-      <!-- Row 3 -->
-      <div>
-        <div class="d-inline-flex justify-content-center py-2 border-bottom">
-          <div>2</div>
-          <div class="div-circle white mx-3"></div>
-          <div class="div-circle black"></div>
-        </div>
-      </div>
-
-      <!-- Row 4 -->
-      <div>
-        <div class="d-inline-flex justify-content-center py-2 border-bottom">
-          <div>3</div>
-          <div class="div-circle black mx-3"></div>
-          <div class="div-circle white"></div>
-        </div>
-      </div>
-
-      <!-- Row 5 -->
-      <div>
-        <div class="d-inline-flex justify-content-center py-2">
-          <div>4</div>
-          <div class="div-circle black mx-3"></div>
-          <div class="div-circle black"></div>
-        </div>
-      </div>
+    <!-- Heading and Description -->
+    <div class="col-12">
+      <h3>{% trans "Dot Combinations" %}</h3>
+    </div>
+    <div class="col-12 mb-3">
+      {% trans 'Click on a dot to flip it. Try to find all the combinations.' %}
+    </div>
+    <div class="form-group col-12 col-sm-4 offset-sm-4">
+      <label for="number-of-dots">{% trans 'Number of dots:' %}</label>
+      <select class="form-control-md" id="number-of-dots" value="3">
```

Comment:
Really strange but only form-control-lg and form-control-sm exist, I think `form-control` is just medium by default.

Reply:
I don't know how but `form-control-md` changed the input box from a standard input dropdown to something I really liked (image) so i want to keep it that way. Does it show like this on chrome?
![image](https://user-images.githubusercontent.com/25916475/59233523-ea662580-8c3c-11e9-8091-fc9ebe422950.png)

Reply:
Unfortunately it shows like this:

![image](https://user-images.githubusercontent.com/16066822/59234262-1505ad80-8c40-11e9-82c5-59304a1c3f92.png)